modules/installation-configuration-parameters.adoc

```diff
@@ -3501,6 +3501,13 @@ For more information on usage, see "Configuring a failure domain" in "Installing
 |The password for the Prism Central user name.
 |String
 
+|platform:
+  nutanix:
+    prismCentral:
+      preloadedOSImageName:
```

Comment:
From the dev PR, it looks like `prismCentral` and `preloadedOSImageName` are nested at the same depth, so likely it is:
```suggestion
    preloadedOSImageName:
```

based on these lines in the dev PR:
```
                  preloadedOSImageName:
                    description: PreloadedOSImageName uses the named preloaded RHCOS
                      image from PC/PE, instead of create and upload a new image for
                      each cluster.
                    type: string
                  prismCentral:
                    description: PrismCentral is the endpoint (address and port) and
                      credentials to connect to the Prism Central. This serves as
                      the default Prism-Central.
```